backend/backend/__main__.py

```diff
@@ -1,35 +1,30 @@
-#import backend.container as container
-import docker
+import backend.container as container
 import time
 
-client = docker.from_env()
-
 print("Hello World")
 
+# Starting ID and port
+ID = 0
+Port = 2222
 
-env = ["PUID=1000", "PGID=1000", "TZ=Europe/London", "SUDO_ACCESS=true",
-       "PASSWORD_ACCESS=true", "USER_PASSWORD=password", "USER_NAME=user"]
-
-ID = "johan"
-port = "2222"
-
-client.containers.run("ghcr.io/linuxserver/openssh-server",
-                      environment=["PUID=1000", "PGID=1000", "TZ=Europe/London", "SUDO_ACCESS=true",
-                                   "PASSWORD_ACCESS=true", "USER_PASSWORD=password", "USER_NAME=user"], hostname="Dell-T140", name="openssh-server" + ID, ports={"2222/tcp": port}, detach=True)
-
-
-time.sleep(15)
+containerHandler = container.Containers()
 
-c = client.containers.get("openssh-server" + ID)
+# Example code for showing multiple containers started and stopped
 
+for i in range(5):
+    containerHandler.create_container(ID, Port, "user", "password")
+    ID += 1
+    Port += 1
 
-def test(input: float) -> int:
-    """[summary]
+time.sleep(3)
 
-    :param input: [description]
-    :type input: float
-    :return: [description]
-    :rtype:
-    """
+ID = 0
+for i in range(5):
+    try:
+        containerHandler.stop_container(ID)
+        containerHandler.destroy_container(ID)
+    except:
+        print("Could not find or stop the specified container, continuing anyways")
+    ID += 1
 
-    return 0
+print("Goodbye World")
```

Comment:
You may want to use the logging library instead, check https://docs.python.org/3/library/logging.html

Reply:
We are aware of this and have now started working on using the logging library you recommended.
This will be part of another user story, namely one addressing the backend "controller" container (the one that runs this custom software, this user story is about getting the command execution docker image up and running with multiple instances).